scss/_patterns_lists.scss

```diff
@@ -186,14 +187,22 @@ $spv-list-item--inner: null;
           width: $bullet-width;
         }
       }
+    }
 
-      & + .p-stepped-list__content {
-        @media (max-width: $breakpoint-heading-threshold) {
-          margin-left: $bullet-width-mobile + $sph-inner;
-        }
+    h#{$i}.p-stepped-list__title + .p-stepped-list__content {
+      @media (max-width: $breakpoint-heading-threshold) {
+        margin-left: $bullet-width-mobile + $sph-inner;
+      }
 
-        @media (min-width: $breakpoint-heading-threshold) {
-          margin-left: $bullet-width + $sph-inner;
+      @media (min-width: $breakpoint-heading-threshold) {
+        margin-left: $bullet-width + $sph-inner;
+      }
+    }
+
+    .p-stepped-list__title {
+      .p-stepped-list--detailed & + .p-stepped-list__content {
```

Comment:
This feels unnecessarily nested.

The only thing within `.p-stepped-list__title` is `.p-stepped-list--detailed & + .p-stepped-list__content` - so, couldn't it be just `.p-stepped-list--detailed .p-stepped-list__title + .p-stepped-list__content` without nesting?